### DISCUSSION TITLE
thumbfast HDR 支持

### DISCUSSION THREAD

**Discussion by ghost** [hooke007/mpv_PlayKit#223] [mpv 通用]

**Comment by hooke007** [comment]
不支持
这是上游的问题

杜比视界仅在 vo=gpu-next 存在有限支持
缩略图编码用的是 vo=lavc

**Reply by hooke007** [reply]
测试 https://github.com/hooke007/MPV_lazy/commit/a86fc6cbaf188c1821aaeb07332b5e2a1f99dfc0
标记 https://github.com/mpv-player/mpv/issues/10678 https://github.com/mpv-player/mpv/issues/11640
